pallets/rewards/rpc/src/lib.rs

```diff
@@ -0,0 +1,71 @@
+use std::sync::Arc;
+
+use jsonrpsee::{
+	core::{async_trait, Error as JsonRpseeError, RpcResult},
+	proc_macros::rpc,
+	types::error::{CallError, ErrorObject},
+};
+pub use pallet_rewards_runtime_api::PolkadexRewardsRuntimeApi;
+use parity_scale_codec::Codec;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+
+const RUNTIME_ERROR: i32 = 1;
+
+#[rpc(client, server)]
+pub trait PolkadexRewardsRpcApi<BlockHash, AccountId, Hash> {
+	#[method(name = "rewards_accountinfo")]
```

Comment:
could be `accountinfo` separated by `_`? e.g. `rewards_account_info`

Reply:
Using the same pattern as `Substrate` using `rewards_accountInfo`